servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -130,4 +153,17 @@ quarkus.container-image.name=nessie
 %test.quarkus.security.users.embedded.roles.test_user=test123
 %test.nessie.version.store.jgit.type=INMEMORY
 
+## test overrides for authorization rule tests
```

Comment:
Move these into `NessieAuthorizationTestProfile` ?

Reply:
the idea for keeping them here is to also have some kind of documentation for rules that we use in testing.